packages/cli/src/commands/projects/init.ts

```diff
@@ -0,0 +1,63 @@
+import { getGitRepoName, readConfig } from "@allurereport/core";
+import { AllureService } from "@allurereport/service";
+import { exit } from "node:process";
+import prompts from "prompts";
+import { green, red } from "yoctocolors";
+import { createCommand } from "../../utils/commands.js";
+
+export const ProjectInitCommandAction = async () => {
```

Comment:
Does the `init` command do something besides creating a project? If not, WDYT about moving the logic into the `project-create`? In other words, make `project-create` retrieve the name from three sources: the CLI, the Repository, and the Prompt. Seems reasonable to me.

Reply:
Yes, it's pretty reasonable

Reply:
Done